### PR TITLE
Feat/activity model

### PR DESCRIPTION
**Pull request by kihifung**
add model: activity and activity_user
Connection between activity, user, and activity_user

### CODE REVIEW

**Review by samliuforwork — APPROVED** [review]
checked

**Review by xx4673 — APPROVED** [review]
checked

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  has_many :activities, through: :activity_users
```
Comment (annielin28815) [inline app/models/user.rb:2]: 記得加has_many: activity_users

**Review by annielin28815 — APPROVED** [review]
good

**Pull request merged**